packages/mobile/src/navigation/welcome/welcome.navigation.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {
+  createNativeStackNavigator,
+  NativeStackNavigationOptions,
+} from '@react-navigation/native-stack';
+import { RootScreenName } from '~/common/enums/enums';
+import { RootNavigationParamList } from '~/common/types/types';
+import { Auth, Welcome } from '~/screens/screens';
+
+const WelcomeNavigation = () => {
+  const NativeStack = createNativeStackNavigator<RootNavigationParamList>();
+  const welcomeScreenOptions: NativeStackNavigationOptions = {
+    headerShown: false,
+  };
+
+  return (
+    <NativeStack.Navigator screenOptions={welcomeScreenOptions}>
+      <NativeStack.Group>
```

Comment:
You can skip using group here as there is only one

Reply:
fixed